csharp/src/Ice/IceSSL/SSLEngine.cs

```diff
@@ -513,75 +326,7 @@ internal void verifyPeer(string address, IceSSL.ConnectionInfo info, string desc
                     _logger.trace(_securityTraceCategory, msg);
                 }
 
-                Ice.SecurityException ex = new Ice.SecurityException();
-                ex.reason = "IceSSL: " + msg;
-                throw ex;
-            }
-
-            if(_verifier != null && !_verifier.verify(info))
-            {
-                string msg = (info.incoming ? "incoming" : "outgoing") +
-                    " connection rejected by certificate verifier\n" + desc;
-                if(_securityTraceLevel >= 1)
-                {
-                    _logger.trace(_securityTraceCategory, msg);
-                }
-
-                Ice.SecurityException ex = new Ice.SecurityException();
-                ex.reason = "IceSSL: " + msg;
-                throw ex;
-            }
-        }
-
-        //
-        // Parse a string of the form "location.name" into two parts.
-        //
-        private static void parseStore(string prop, string store, ref StoreLocation loc, ref StoreName name,
```

Comment:
Used only to parse `IceSSL.FindCert.*` deprecated property (now removed). `IceSSL.FindCert` is still supported.